### ISSUE TITLE
writers.las improperly reads pipeline string option containing space

### ISSUE THREAD

**Issue by n4z4m3** [PDAL/PDAL#3964]
**Description of bug**
When a LAS file System Identifier or Generating Software contains spaces the writer strips everything after the space for some reason.
For example, if I have a Ssytem Identifier of "My System ID", it will be written out as "My". And if it starts with a space, it will end up with an empty string.

**Expected behavior**
The System Identifier and Generating Software values should not be modified by the writer except to maybe truncate them to 32 characters since that is the maximum allowed by the LAS file format.

**Comment by abellgithub** [comment]
Please provide the pipeline/command you're using and input file.

**Comment by n4z4m3** [comment]
I have attached a sample input LAS file and a Pipeline JSON file which is able to replicate the issue.

Just run 
`pdal pipeline test_writer.json`

The output sample_out.json will have software_id of "My" when it should  be "My Software ID" and the system_id will also be "My" when it should be "My System ID"

This is when running using the command line version of pdal in Windows (pdal.exe). 
I have also tried through python code using the conda version. This is with the latest release of PDAL. (2.5.0)
I have also tried in Red Hat Linux with the same results.
This is why I did not mention the system in my bug report because it does not appear to matter. It appears to happen regardless of the system.
